src/messaging/classes/Message.ts

```diff
@@ -50,72 +54,157 @@ export class Message extends HttpClient {
+                                        minItems: 1,
+                                        maxItems: 10,
+                                        uniqueItems: true,
+                                    },
+                                ],
+                            },
+                            from: {
+                                type: 'string',
+                            },
+                            notifyURL: {
+                                type: 'string',
+                            },
+                            replyRequest: {
+                                type: 'boolean',
+                            },
+                            subject: {
+                                type: 'string',
+                                maxLength: 64,
+                            },
+                            multimedia: {
+                                type: 'object',
+                                properties: {
+                                    type: {
+                                        type: 'string',
+                                        enum: [
+                                            TMultimediaMessageType.AUDIO_AMR,
+                                            TMultimediaMessageType.AUDIO_AAC,
+                                            TMultimediaMessageType.AUDIO_MP3,
+                                            TMultimediaMessageType.AUDIO_MPEG3,
+                                            TMultimediaMessageType.AUDIO_MPEG,
+                                            TMultimediaMessageType.AUDIO_MPG,
+                                            TMultimediaMessageType.AUDIO_WAV,
+                                            TMultimediaMessageType.AUDIO_3GPP,
+                                            TMultimediaMessageType.AUDIO_MP4,
+                                            TMultimediaMessageType.IMAGE_GIF,
+                                            TMultimediaMessageType.IMAGE_JPEG,
+                                            TMultimediaMessageType.IMAGE_JPG,
+                                            TMultimediaMessageType.IMAGE_PNG,
+                                            TMultimediaMessageType.IMAGE_BMP,
+                                            TMultimediaMessageType.VIDEO_MPEG4,
+                                            TMultimediaMessageType.VIDEO_MP4,
+                                            TMultimediaMessageType.VIDEO_MPEG,
+                                            TMultimediaMessageType.VIDEO_3GPP,
+                                            TMultimediaMessageType.VIDEO_3GP,
+                                            TMultimediaMessageType.VIDEO_H263,
+                                            TMultimediaMessageType.TEXT_PLAIN,
+                                            TMultimediaMessageType.TEXT_X_VCARD,
+                                            TMultimediaMessageType.TEXT_X_VCALENDAR,
```

Comment:
@AnanditaBhatt-tdev as discussed, the inline schemas will be moved from the libraries to the openapi specs as constraints in each definition file, if you see value in leaving these enums as type definitions feel free to keep them there or convert to JSON schema enums.